components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/OAuth2Service.java

```diff
@@ -981,7 +985,24 @@ public Claim[] getUserClaims(String accessTokenIdentifier) {
     public String getOauthApplicationState(String consumerKey) {
 
         try {
-            OAuthAppDO appDO = OAuth2Util.getAppInformationByClientId(consumerKey);
+            String tenantDomain = IdentityTenantUtil.getTenantDomain(IdentityTenantUtil.getLoginTenantId());
```

Comment:
If the path is /t/tenant/.../token what will be resolved as the tenantDomain?
if the path is /t/tenant/o/org.../token what will be resolved as the tenantDomain?